.github/workflows/go.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.19
+        go-version: 1.21
```

Comment:
Maybe it is better to do this in a separate issue? Or create an issue for upgrading to go 1.21 and mention it here? Also why not go 1.23?